quantum/quantum_keycodes.h

```diff
@@ -216,4 +216,37 @@
 #define SQ_R(n) (n < SEQUENCER_RESOLUTIONS ? SEQUENCER_RESOLUTION_MIN + n : KC_NO)
 #define SQ_T(n) (n < SEQUENCER_TRACKS ? SEQUENCER_TRACK_MIN + n : KC_NO)
 
+// Pointing device mode keys 16 modes max
+// pointing device mode key macros
+#define POINTING_MODE_COUNT ((QK_POINTING_MODE_MAX - QK_POINTING_MODE + 1) / 2)
+// Momentary scroll mode
+#define PM_MO(pm) (MIN((pm), (POINTING_MODE_COUNT - 1)) + QK_POINTING_MODE)
+// Toggle default scroll mode
+#define PM_TG(pm) (MIN((pm), (POINTING_MODE_COUNT - 1)) + QK_POINTING_MODE + POINTING_MODE_COUNT)
+
+// Default Pointing device pointing modes
+enum pointing_device_mode_list {
+    PM_NONE,
+    PM_PRECISION,
+    PM_DRAG,
+    PM_CARET,
+    PM_HISTORY,
+    PM_VOLUME,
+// safe range for custom modes with built in keycodes
+#ifndef EXTRAKEY_ENABLE
+    PM_SAFE_RANGE = PM_VOLUME,
+#else
+    PM_SAFE_RANGE,
+#endif
```

Comment:
Shouldn't be conditional.  Should be okay to have a "dead" mode, since they can be overwritten, anyways.  That or the volume option should be removed and delegated to user 
```suggestion
    PM_SAFE_RANGE,
```

Reply:
yeah I'll just put it back so at least the maps will default to overwriting the spot.